src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -94,7 +101,21 @@ object ErrorAggregator {
     .add[Int]("content_shutdown_crashes")
     .build
 
-  private val statsSchema = SchemaBuilder.merge(metricsSchema, countHistogramErrorsSchema)
+  private val thresholdHistogramsSchema = thresholdHistograms.foldLeft(new SchemaBuilder())( (schema, kv) => {
+    val histogramName = kv._1
+    kv._2 match {
+      case (processTypes: List[String], thresholds: List[Int]) => {
+        val fields = for {
+          processType <- processTypes
+          threshold <- thresholds
+        } yield s"${histogramName}_${processType}_${threshold}"
```

Comment:
Should be `s"${histogramName}_${processType}_above_${threshold}"`, and all lowercase.